src/main/java/org/reactome/qa/doisuggester/ReactionlikeEvent.java

```diff
@@ -0,0 +1,354 @@
+package org.reactome.qa.doisuggester;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+
+/**
+ * Represents a Reactionlike Event, in the context of DOI suggestion activities.
+ * @author sshorser
+ *
+ */
+public class ReactionlikeEvent
+{
+	// The underlying RLE, in the current database.
+	private GKInstance rle;
+	// Corresponding RLE from the previous Release's database.
+	private ReactionlikeEvent prevRle = null;
+
+	private List<GKInstance> authoredInstancedEdits;
+	private List<GKInstance> reviewedInstancedEdits;
+	private List<GKInstance> revisedInstancedEdits;
+
+	private Set<GKInstance> ancestors = new HashSet<>();
+
+	/**
+	 * Constructor to create a DOISuggester-contexted Reaction-like Event object.
+	 * @param instance - The underlying instance from the database. This is a ReactionlikeEvent object.
+	 * @param prevAdaptor - A database adaptor that should point to the *previous* version of the database. This will be used to look at
+	 * previous versions of the <code>instance</code>
+	 * @throws Exception
```

Comment:
The exception is from the dbAdaptor, I guess?  A note describing when this method throws an exception would be good.

Reply:
Maybe something like `@throws Exception Thrown when unable to fetch corresponding instance from previous release database`

Reply:
Yes, this was done in 3768088

Reply:
Yeah, I wrote that as you were committing the changes!  Thanks - the message added looks good.